account_ebics/__manifest__.py

```diff
@@ -18,6 +18,7 @@
         "views/ebics_userid_views.xml",
         "views/ebics_file_format_views.xml",
         "wizards/ebics_change_passphrase.xml",
+        "wizards/ebics_admin_order.xml",
         "wizards/ebics_xfer.xml",
```

Comment:
Hi @luc-demeyer hope you are well :slightly_smiling_face: , I am having problems installing account_ebics in a new empty database, the error says:

raise ValueError('External ID not found in the system: %s' % xmlid)
ValueError: External ID not found in the system: account_ebics.ebics_xfer_view_form_download.

From the first view in the file account_ebics/wizards/ebics_admin_order.xml:4

I changed locally the order of these 2 manifest lines and was able to install it.

The first view in the ebics_admin_order.xml inherits a view in the file ebics_xfer.xml, so in this case the order of the files needs to be changed IMO.

I was not sure if I should comment this PR, open an issue or even create a PR for the changes.
Please let me know if I should do something else instead.

Regards.

Reply:
@BT-cjimeno 
Thanks a lot for this valuable feedback.
I implemented your correction, cf. https://github.com/Noviat/account_ebics/pull/97/commits/970bee6c7f7c3e12a1d7b5c55274a5ef8ff700cc

Concerning the way to report such issues : in order of prefence
- make PR with fix
- Open an Issue
- comment the PR that caused the problem